app/javascript/dashboard/components/widgets/conversation/ReplyBox.vue

```diff
@@ -629,7 +630,7 @@ export default {
       if (this.conversationIdByRoute) {
         const key = `draft-${this.conversationIdByRoute}-${this.replyType}`;
         const messageFromStore =
-          this.$store.getters['draftMessages/get'](key) || '';
+          this.$store.getters['draftMessages/get'](key) || this.currentChat.draft_message_content || '';
```

Comment:
Same comment as above. We should not be polluting the conversation namespace rather consider the draftMessage as an entity by itself.